spec/features/logging/view_auth_requests_for_a_username_spec.rb

```diff
@@ -1,4 +1,53 @@
-describe "View authentication requests for a username", type: :feature do
+describe "View authentication requests for a username",focus: true, type: :feature do
+
+  context "with super admin privileages" do
+    let(:username) { "AAAAAA" }
+    let(:search_string) { "AAAAAA" }
+    let(:organisation) { create(:organisation) }
+    let!(:super_admin) { create(:user, super_admin: true) }
+    let(:location) { create(:location, organisation_id: organisation.id) }
+
+    before do
+      Session.create!(
+        start: 3.days.ago,
+        username: username,
+        mac: "",
+        ap: "",
+        siteIP: "1.1.1.1",
+        success: true,
+        building_identifier: ""
+      )
+      Session.create!(
+        start: 3.days.ago,
+        username: username,
+        mac: "",
+        ap: "",
+        siteIP: "1.1.1.1",
+        success: true,
+        building_identifier: ""
+      )
+
+      organisation = create(:organisation)
+      create(:ip, location_id: location.id, address: "1.1.1.1")
+
+      sign_in_user super_admin
+      visit new_logs_search_path
+      choose "Username"
+      click_on "Go to search"
+      fill_in "Username", with: search_string
+      click_on "Show logs"
+    end
+
+    context "when username is correct" do
+      let(:search_string) { "AAAAAA" }
+
+      it "displays two results" do
+        expect(page).to have_content("Found 2 results for \"#{username}\"")
+      end
+
+    end
+  end
```

Comment:
This test looks pretty good. It could be improved by making it really obvious that the super admin is seeing logs for multiple organisations that they do not belong to.

For instance:
- create the super admin that belongs to org 1.
- create another organisation with a location, ip and session.
- create a third organisation with a location, ip and session

Show that the super admin can see all the logs of all the organisations.

Could be something like:
```
context "with super admin privileages"
  context "with many sessions belonging to many organisations"
    it "displays all the sessions to the super admin"
```

Reply:
Also worth double checking that the normal admin side of this is properly tested. A normal admin should definitely not be able to see any other organisations logs.